specification/hugr.md

```diff
@@ -1085,16 +1044,7 @@ running different resources. By the same mechanism, Tierkreis can reason
 about where to run different parts of the graph by inspecting their
 resource requirements.
 
-### Type Constraints
-
-We will likely also want to add a fixed set of attributes to certain
-subsets of `TYPE`. In Tierkreis these are called “type constraints”. For
-example, a potential `HashMap` type can only be constructed when the type that we
-map from is `Hashable`. For the Hugr, we will need this `Hashable`
-constraint, as well as a `Classic` constraint.
-
-
-### Resources
+<!--### Resources
```

Comment:
This seemed to duplicate what was written earlier in the "Extension Implementation" section....shall I remove altogether?